Userland/Libraries/LibWeb/CSS/Parser/DeclarationOrAtRule.h

```diff
@@ -7,6 +7,7 @@
 #pragma once
 
 #include <LibWeb/CSS/Parser/Declaration.h>
+#include <LibWeb/CSS/Parser/Function.h>
```

Comment:
Why's this one needed? `Function` doesn't appear in this header

Reply:
You're right, thanks! I intended to add it to the `.cpp` file.

In case you're curious: DeclarationOrAtRule contains a Declaration contains a ComponentValue contains a Function. So when the destructor is defined in `DeclarationOrAtRule.cpp`, the full definition of `Function` is necessary.